test/integration/agent/route_test.go

```diff
@@ -232,6 +231,23 @@ func TestInitialize(t *testing.T) {
 			assert.Equal(t, expectedData, string(actualData), "mismatch iptables data in table %s", table)
 		}
 	}
+	t.Log("Testing ipTables syncLoop")
+	deleteOption := fmt.Sprintf("iptables -t %s -D %s ", iptables.RawTable, iptables.PreRoutingChain)
+	deleteRuleCmd := deleteOption + strings.Join([]string{"-m", "comment", "--comment", "\"Antrea: jump to Antrea prerouting rules\"", "-j", "ANTREA-PREROUTING"}, " ")
+
+	// #nosec G204: ignore in test code
+	actualData, err := exec.Command("bash", "-c", deleteRuleCmd).Output()
```

Comment:
I notice other tests doing the same thing (calling `iptables` through `bash`), but I wonder why the `iptables` executable cannot be invoked directly instead?